README.md

```diff
@@ -1,2 +1,429 @@
+                     Optional selective set of identifiers to register. If not specified, all the tables will be registered. Use this when there are
+                       few identifiers that need to be registered. For a large number of identifiers, use the `--identifiers-from-file` or
+                       `--identifiers-regex` option.
+                     Example: --identifiers foo.t1,bar.t2
+      --identifiers-from-file=<identifiersFromFile>
+                     Optional text file path that contains a set of table identifiers (one per line) to register. Should not be used with
+                       `--identifiers` or `--identifiers-regex` option.
+                     Example: --identifiers-from-file /tmp/files/ids.txt
+      --identifiers-regex=<identifiersRegEx>
+                     Optional regular expression pattern used to register only the tables whose identifiers match this pattern. Should not be used
+                       with `--identifiers` or '--identifiers-from-file' option.
+                     Example: --identifiers-regex ^foo\..*
+```
+
+Note: Options for migrate command is exactly same as register command.
+
+> :warning: **It is recommended to use this CLI tool when there is no in-progress commits for the tables in the source catalog.**
+In-progress commits may not make it into the target catalog if used.
+
+> :warning: `register` command just registers the table. 
+Which means the table will be present in both the catalogs after registering.
+Operating same table from more than one catalog can lead to missing updates, loss of data and table corruption. 
+So, it is recommended to use the 'migrate' command in CLI to automatically delete the table from source catalog after registering 
+or avoid operating tables from the source catalog after registering if 'migrate' command is not used.
+
+# Sample Inputs
+## Bulk migrating all the tables from Hadoop catalog to Nessie catalog (main branch)
+```shell
+java -jar iceberg-catalog-migrator-cli-0.1.0-SNAPSHOT.jar migrate \
+--source-catalog-type HADOOP \
+--source-catalog-properties warehouse=/tmp/warehouse,type=hadoop \
+--target-catalog-type NESSIE  \
+--target-catalog-properties uri=http://localhost:19120/api/v1,ref=main,warehouse=/tmp/warehouse
+```
+
+## Register all the tables from Hadoop catalog to Arctic catalog (main branch)
+
+```shell
+export PAT=xxxxxxx
+export SECRETKEY=xxxxxxx
+export ACCESSKEY=xxxxxxx
+```
+
+```shell
+java -jar iceberg-catalog-migrator-cli-0.1.0-SNAPSHOT.jar register \
+--source-catalog-type HADOOP \
+--source-catalog-properties warehouse=/tmp/warehouse,type=hadoop \
+--target-catalog-type NESSIE \
+--target-catalog-properties uri=https://nessie.test1.dremio.site/v1/repositories/8158e68a-5046-42c6-a7e4-c920d9ae2475,ref=main,warehouse=/tmp/warehouse,authentication.type=BEARER,authentication.token=$PAT \
+--target-catalog-hadoop-conf fs.s3a.secret.key=$SECRETKEY,fs.s3a.access.key=$ACCESSKEY
```

Comment:
Don't these options support passing the keys via environment variables?
Definitely mention that passing secrets and other credentials on the command line is NOT safe

Reply:
updated.

Reply:
Really? Nothing changed here.

Reply:
removed mention of keys in the catalog options descriptions and in README as it works without it. 

I don't want to remove `catalog-hadoop-conf` option itself because I am not sure which catalogs uses it. 
Iceberg also exposes hadoop conf for catalog configurations. So, I had to expose it. 

https://github.com/apache/iceberg/blob/b756a5da3559be9deee9cdc674c4bf36ac291231/core/src/main/java/org/apache/iceberg/CatalogUtil.java#L216